Scripts/script-StixCreator.yml

```diff
@@ -0,0 +1,118 @@
+commonfields:
+  id: StixCreator
+  version: -1
+name: StixCreator
+script: |-
+  ''' IMPORTS '''
+  import json
+
+  from stix2 import Indicator, Bundle
+
+  indicators = []
+
+  userArgs = demisto.args().get('indicators','Unknown')
+  if isinstance(userArgs, dict) == True:
+      all_args = json.loads(json.dumps(userArgs))
+  else:
+      all_args = json.loads(demisto.args().get('indicators','Unknown'))
```

Comment:
json.loads will raise an Exception if string passed to it. So either replace 'Unknown' with dict or change the whole error logic

Reply:
not sure exactly what you meant here I would like to talk with you about it tomorrow